.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Reset NX cache
         run: yarn nx reset
       - name: Compress artifact
-        run: tar -cf install-build-node-${{ matrix.node-version }}.tar ./packages/core/lib ./packages/validator-js/lib ./packages/utils/lib ./node_modules
+        run: tar -cf install-build-node-${{ matrix.node-version }}.tar ./packages/core/lib ./packages/validator-js/lib ./packages/utils/lib ./packages/postgres/lib ./node_modules
       - uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
```

Comment:
I can't comment on line 88, but we'll need to run unit tests from the postgres library as well

Reply:
Done & Added utils

Lerna/NX has something to only run tests of things that were impacted by the PR. Could be worth looking into

Reply:
Not sure if I want that for CI, but for local development it would be nice to set up indeed. Or maybe we can do something hybrid; when a PR is open for review it only runs affected tests and when it's approved/auto-merge is enabled the entire test suite runs before merging. Then the amount of resources used for each new commit is more limited but we still have the safety of the entire test suite having passed

Reply:
If it's configured properly, I don't see any risk to be honest. Lerna knows which packages depend on what, so it would skip tests on packages that don't depend on anything if they're not modified, but modifying something like `utils` would run the test suites of all packages

Only considering unit tests too, not integration tests at the moment

It would also mean all unit tests are a single task in our CI, but which one fails is still readable thanks to nx collapsing the results per package in the output:

![image](https://github.com/sequelize/sequelize/assets/1280915/d8170f51-33a6-4645-af54-b7b585a406b9)

Reply:
Let's try it out with unit/typing tests then. And indeed with splitting to the separate packages we don't need the separate steps. It was mainly that if the unit tests for core failed you couldn't easily see for which dialect it failed, which we can solve with running the tests from the dialect package

Reply:
OK! Let's try in a separate PR. I added it to the list of future improvements